src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasSlicer.java

```diff
@@ -1483,15 +1403,16 @@ private void sliceMultiPolygonRelation(final CompleteRelation relation) // NOSON
         final Map<String, CompleteRelation> newRelations = new HashMap<>();
         clippedMultiPolygons.keySet().forEach(countryCode ->
         {
-            final long newRelationId = relationIdentifierFactory.nextIdentifier();
-            newRelationIds.add(newRelationId);
-            if (this.getCountries().contains(countryCode))
+            final CompleteRelation newRelation = CompleteRelation.shallowFrom(relation)
+                    .withIdentifier(relationIdentifierFactory.nextIdentifier())
+                    .withTags(relation.getTags())
+                    .withAddedTag(SyntheticGeometrySlicedTag.KEY,
```

Comment:
Should there be a toString here too?

Reply:
no-- the key is a private member that is already a string, the YES is the enum value so `toString()` has to be explicitly invoked